docs/guides/publisher-client-side.md

```diff
@@ -231,29 +239,31 @@ The UID2 SDK encrypts the hash before sending it to the UID2 service.
+The format of the object stored in local storage could change without notice. We recommend that you do **not** read and update the object in local storage directly. 
 :::
 
 ## Example Integration Codes and When to Pass DII to the UID2 SDK
 
-When this is the first page load with no identity (TODO link to glossary!), you will need to call one of the aforementioned ```setIdentity``` methods with a DII to start the token generation call. Once an identity (TODO link to glossary!) is generated, the advertising token (TODO link to glossary!) you would send to bidstream will be available by waiting for the ```IdentityUpdated``` event from the SDK, for example, see how the value for ```advertising_token_to_use``` is set in the code snippet below.
+When this is the first page load with no [identity](../ref-info/glossary-uid.md#gl-identity), to start the token generation call you'll need to call one of the `setIdentity` methods with DII. Once an identity is generated, the advertising token ([UID2 token](../ref-info/glossary-uid.md#gl-uid2-token)) that you would send to the bid stream will be available by waiting for the `IdentityUpdated` event from the SDK. For an example, see how the value for `advertising_token_to_use` is set in the following code snippet.
 
 In some cases, the user's DII is not available on page load, and getting the DII has some associated cost. For example, an API call might be required to fetch the DII, or the user has to be prompted to provide the DII information.
 
-You can potentially avoid that cost by checking for an existing token that you can use or refresh - simply calling
-[__uid2.isLoginRequired](../sdks/client-side-identity#isloginrequired-boolean) which returns a boolean and if it returns true, this means UID2 SDK is unable to create a new advertising token with the existing resource and a DII is required to generate a brand new UID2 token.
+You can potentially avoid that cost by checking for an existing token that you can use or refresh. To do this, call
+[__uid2.isLoginRequired](../sdks/client-side-identity#isloginrequired-boolean) which returns a Boolean value. If it returns `true`, this means that the UID2 SDK cannot create a new advertising token with the existing resource and DII is required to generate a brand new UID2 token.
+
+The following code snippets demonstrate how you might integrate with the UID2 SDK for JavaScript for the two scenarios above&#8212;starting with no token as well as reusing/refreshing any existing UID2 token if found. 
 
-The following code snippets demonstrate how you might integrate with UID2 SDK for JavaScript for the two scenarios above - starts with no token as well as reusing/refresh any existing UID2 token if found. 
+{**GWH_SW_03 is there more than one snippet? Looks like one to me. Could we break it up and have a text intro for each piece? Maybe not a good idea. Just checking**}
```

Comment:
_{**GWH_SW_03 is there more than one snippet? Looks like one to me. Could we break it up and have a text intro for each piece? Maybe not a good idea. Just checking**}_

yeh only 1 snippet. i think a single block is fine so readers can have a holistic view on it

Reply:
Thx. Changed the intro copy from "The following code snippets demonstrate..." to The following code snippet demonstrates...". Removed my query.